tests/post_training/pipelines/causal_language_model.py

```diff
@@ -40,6 +40,8 @@ def transform_func(examples):
         return transform_func
 
     def prepare_calibration_dataset(self):
+        if self.batch_size > 1:
+            print("Batch size > 1 is not supported for causal language models. Batch size = 1 will be used.")
```

Comment:
Where is the `self.batch_size = 1` setting then?

Reply:
done